android/build.gradle

```diff
@@ -36,6 +36,8 @@ android {
 }
 
 repositories {
+    google()
```

Comment:
Why were these added?

Reply:
I had a build error "Could not find com.android.tools.build:aapt2:3.3.0-5013011".

Reply:
That explains `google`, but why jcenter?

Reply:
After fixing  "Could not find com.android.tools.build:aapt2:3.3.0-5013011" by adding `google()`, there is another artifact missing `Could not find org.jetbrains.trove4j:trove4j`.